buildout.cfg

```diff
@@ -37,7 +37,7 @@ zope.testrunner = 4.4.3
 [script]
 recipe = zc.recipe.egg
 dependent-scripts = true
-interpreter = python
```

Comment:
Nope you're right, this is an unrelated change. I added it because I have a habit of using virtualenvs even when using buildout and in this case, buildout tries to overwrite the python interpreter created by virtualenv which breaks things. Happy to change it back though and keep it as a local modification.